pkg/che/che.go

```diff
@@ -120,6 +120,24 @@ func (c *Client) DeleteUser(userID string) error {
 	return err
 }
 
+// UserAPICheck returns an error if the Che user API cannot be reached successfully, returns nil otherwise
+func (c *Client) UserAPICheck() error {
```

Comment:
Any tests for this method?

Reply:
Ah yes, added in 128cdab96a227412f60ea41804ed4a7e53106211